main.cpp

```diff
@@ -187,6 +249,7 @@ void run()
     << std::left << std::setw(12) << "Average" << std::endl;
 
   std::cout << std::fixed;
+  }
```

Comment:
This is fine, but might be interesting to see these results per rank *as well* as the aggregate bandwidth. This would make it easier to see how well things scale when running on lots of nodes.